src/ui/public/stringify/types/color_default.js

```diff
@@ -0,0 +1,6 @@
+export const DEFAULT_COLOR = {
+  range: `${Number.NEGATIVE_INFINITY}:${Number.POSITIVE_INFINITY}`,
+  regex: '<insert regex>',
```

Comment:
__question:__ There is no need to change anything in this PR, I'm just wondering why we set a custom non-empty string as a default regex and don't use `placeholder` field for `<insert regex>`?